releasenotes/notes/add-ssvqe-algorithm-6c395267cdd03798.yaml

```diff
@@ -0,0 +1,38 @@
+---
+features:
+  - |
+    Adds the Subspace Search Variational Quantum Eigensolver (:class:`SSVQE`)
+    algorithm. SSVQE is a generalization of the Variational Quantum Eigensolver
+    (VQE) which aims to find the low-lying eigenvalue/eigenvector pairs of a
+    Hermitian operator :math:`H`. It takes a set of mutually orthogonal input
+    states, applies a parameterized ansatz (:class:`QuantumCircuit`) to all of
+    them, then minimizes a weighted sum of the expectation values of :math:`H` 
+    with respect to these parameterized states using a classical optimizer
+    (:class:`qiskit.algorithms.optimizers.Optimizer`).
+    For example:
+
+    .. code-block:: python
+
+      from qiskit import QuantumCircuit
+      from qiskit.opflow import Z
```

Comment:
We try to limit the use of opflow now. Could you use `Pauli("ZZ")`?

Reply:
Okay, I have now changed this.